### ISSUE TITLE
[Locale] Move translations to Transifex or Crowdin?

### ISSUE THREAD

**Issue by IBPX** [Tuxemon/Tuxemon#234]
The current system is very clunky and unintuitive, I think, and would be difficult to people who speak another language but are not already used to editing JSON files.

Services such as Transifex (my personal favorite) and Crowdin can be used for translation very simply. Both support JSON files, and if your project is open-source, are free to use. Many many translations of open-source projects are run on Transifex.

What do you think? Yes? No? Do you prefer just plain Git?

**Comment by bitcraft** [comment]
A lot of work has already been done to support translations.  I'm not sure that there is enough motivation right now (I certainly don't have the time) to rework [again] the translation support for gettext.  So far I think the json system has worked well enough, and I'm happy to let it sit for a while.

I'm not really against your proposal, but the current method is the least amount of effort for the maintainers to handle.

If somebody wants to step up and maybe become the leader of translation support, then I could see this going somewhere....providing enough people want to open transifex accounts, learn gettext, etc.

I'm also going to mention that while that game is being translated, there isn't much support for game content (maps, etc) to be translated, so any work done before that is complete, may be creating more work down the road.  For example, we may use the well-known jinja2 library for scripting.

I'm already a bit wary of too much burn out from translators, especially in regards to the TMXN descriptions and what not.  The content producing side of this project may decide to change these things, creating constant churn while the game is made.  I'd really like to reduce the amount of back-and-forth duplication of work that [could] happens here.

Somewhat related:

I'd kinda prefer if we could just let the game be mostly English until we hit a milestone, then let translators work on it.  Before we opened the flood gates for translation support, we had already revised and changed some of the game strings, and I'm sure it will happen again.  The "butterfly effect" applies here...a change to the English translation will cause the other ones to change.

TL;DR.  I don't want to implement it now, but maybe somebody else will.  The game content may change, creating too much work for translators.  These are just my opinions, feel free to discuss.

**Comment by Clonewayx** [comment]
[Translatewiki](https://translatewiki.net/wiki/Translating:New_project) have [support for JSON format.]( https://www.mediawiki.org/wiki/Help:Extension:Translate/File_format_support)

**Comment by IBPX** [comment]
@Clonewayx Hadn't heard of that before.

**Comment by kerizane** [comment]
We are now using weblate.
